autotest/src/main/kotlin/no/nav/familie/tilbake/e2e/domene/builder/TilbakekrevingBuilder.kt

```diff
@@ -0,0 +1,79 @@
+import no.nav.familie.kontrakter.felles.Språkkode
+import no.nav.familie.kontrakter.felles.tilbakekreving.Tilbakekrevingsvalg
+import no.nav.familie.kontrakter.felles.tilbakekreving.Varsel
+import no.nav.familie.kontrakter.felles.tilbakekreving.Verge
+import no.nav.familie.kontrakter.felles.tilbakekreving.Vergetype
+import no.nav.familie.kontrakter.felles.tilbakekreving.Ytelsestype
+import java.math.BigDecimal
+import java.time.LocalDate
+import kotlin.random.Random
+
+class TilbakekrevingBuilder(eksternFagsakId: String,
+                            eksternBehandlingId: String? = null,
+                            fagsystem: Fagsystem,
+                            ytelsestype: Ytelsestype,
+                            varsel: Boolean,
+                            verge: Boolean) {
+
+    private val request = OpprettTilbakekrevingRequest(fagsystem = fagsystem,
+                                                       ytelsestype = ytelsestype,
+                                                       eksternFagsakId = eksternFagsakId,
+                                                       eksternId = eksternBehandlingId ?: Random.nextInt(1000000, 9999999)
+                                                               .toString(),
+                                                       personIdent = "12345678901",
+                                                       saksbehandlerIdent = "Z994824",
+                                                       behandlingstype = Behandlingstype.TILBAKEKREVING,
+                                                       språkkode = Språkkode.NB,
+                                                       enhetId = "0106",
+                                                       enhetsnavn = "NAV Fredrikstad",
+                                                       revurderingsvedtaksdato = LocalDate.now().minusDays(35),
+                                                       faktainfo = Faktainfo(revurderingsårsak = "Nye opplysninger",
+                                                                             revurderingsresultat = "Endring i ytelsen",
+                                                                             tilbakekrevingsvalg = utledTilbakekrevingsvalg(varsel),
+                                                                             konsekvensForYtelser = setOf("Reduksjon av ytelsen",
+                                                                                                          "Feilutbetaling")),
+                                                       varsel = utledVarsel(varsel),
+                                                       manueltOpprettet = false,
+                                                       verge = utledVerge(verge))
+
+    private fun utledTilbakekrevingsvalg(varsel: Boolean): Tilbakekrevingsvalg {
+        return if (varsel) Tilbakekrevingsvalg.OPPRETT_TILBAKEKREVING_MED_VARSEL else Tilbakekrevingsvalg.OPPRETT_TILBAKEKREVING_UTEN_VARSEL
```

Comment:
kommentar er for lang (mer enn 130 tegn)

Reply:
La inn linjeskift før "else"